packages/perseus/src/widgets/__tests__/interactive-graph.test.ts

```diff
@@ -142,56 +141,19 @@ describe("mafs graphs", () => {
         flags: {mafs: graphTypeFlags},
     };
 
-    describe("locked layer", () => {
-        it("should render locked points", async () => {
-            // Arrange
-            const {container} = renderQuestion(
-                segmentWithLockedPointsQuestion,
-                apiOptions,
-            );
-
-            // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
-            const points = container.querySelectorAll(
-                // Filter out the interactive points' circles
-                "circle:not([class*='movable-point'])",
-            );
-
-            // Act
-
-            // Assert
-            expect(points).toHaveLength(2);
-        });
-
-        test("should render locked points with styles", async () => {
-            // Arrange
-            const {container} = renderQuestion(
-                segmentWithLockedPointsQuestion,
-                apiOptions,
-            );
-
-            // Act
-            // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
-            const points = container.querySelectorAll(
-                "circle:not([class*='movable-point'])",
-            );
-
-            // Assert
-            expect(points[0]).toHaveStyle({fill: color.red, stroke: color.red});
-            expect(points[1]).toHaveStyle({fill: color.red, stroke: color.red});
-        });
-    });
-
-    describe("segment", () => {
+    describe.each([
+        segmentQuestionDefaultCorrect,
+        linearQuestionWithDefaultCorrect,
+        linearSystemQuestionWithDefaultCorrect,
+        rayQuestionWithDefaultCorrect,
+    ])("graph type", (question) => {
```

Comment:
**idea**: this isn't a massive issue, but if any of these fail, the test name won't reveal which graph type failed.  

What do you think about this?
```suggestion
        ["segment", segmentQuestionDefaultCorrect],
        ["linear", linearQuestionWithDefaultCorrect],
        ["linear-system", linearSystemQuestionWithDefaultCorrect],
        ["ray", rayQuestionWithDefaultCorrect],
    ])("graph type: %s", (type, question) => {
```

This adds a bit of maintenance overhead but then we know which graph type failed.

Reply:
I was thinking about doing this as well, just stopped because I thought I was being a bit extra. I'll get back to it Monday